client/homebrew/editor/editor.jsx

```diff
@@ -189,7 +190,28 @@ const Editor = createClass({
 	brewJump : function(targetPage=this.getCurrentPage()){
 		if(!window || this.isMeta()) return;
 		// console.log(`Scroll to: p${targetPage}`);
-		window.frames['BrewRenderer'].contentDocument.getElementById(`p${targetPage}`).scrollIntoView({ behavior: 'smooth', block: 'start' });
+		const currentPos = window.frames['BrewRenderer'].contentDocument.getElementsByClassName('brewRenderer')[0].scrollTop;
```

Comment:
Since you access `window.frames['BrewRenderer'].contentDocument.getElementsByClassName('brewRenderer')[0]` multiple times, should it be stored in a `const` ?

Reply:
That's a good idea; I've made the change, tested, and as it's working fine, pushed it through in a fresh commit now.